response_operations_ui/views/messages.py

```diff
@@ -121,14 +121,46 @@ def view_messages():
         return render_template("messages.html", breadcrumbs=breadcrumbs, response_error=True)
 
 
+@messages_bp.route('/threads/<thread_id>', methods=['GET'])
+@login_required
+def view_conversation(thread_id):
+    try:
+        thread_conversation = message_controllers.get_conversation(thread_id)['messages']
+        refined_thread = [_refine(message) for message in reversed(thread_conversation)]
```

Comment:
🎨 Could do `refined_thread = [_refine(message) for message in reversed(message_controllers.get_conversation(thread_id)['messages'])]`

Reply:
@dcdarrell9 i've done it this way in the past but i've been suggested to change it so i'll leave it for now